functional_tests/scylla_operator/test_functional.py

```diff
@@ -15,13 +15,15 @@
 
 import logging
 import random
+import threading
 import time
 import pytest
 
-from sdcm.mgmt import TaskStatus  # pylint: disable=import-error
-from sdcm.utils.k8s import HelmValues  # pylint: disable=import-error
-from functional_tests.scylla_operator.libs.helpers import get_orphaned_services
-
+from sdcm.cluster_k8s import ScyllaPodCluster
+from sdcm.mgmt import TaskStatus
+from sdcm.utils.k8s import HelmValues
+from functional_tests.scylla_operator.libs.helpers import get_orphaned_services, scylla_operator_rollout_restart, \
```

Comment:
Please, use following approach for long imports for better readability:

```
from functional_tests.scylla_operator.libs.helpers import (
    get_orphaned_services,
    scylla_operator_pods_and_statuses,
    scylla_operator_rollout_restart,
    wait_for_scylla_operator_rollout_complete,
)
```

Reply:
done

Reply:
Actually it is not done. It is different than the requested variant.